CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# librdkafka v2.5.1
+
+librdkafka v2.5.1 is a feature release.
+
+* Fix an assert being triggered when no metrics matched on the client side during send push telemetry call.
```

Comment:
Add the `Fixes` section with references to this PR, the issue and that happens in 2.5.0, follow other fixes template.
Add a warning in 2.5.0 release saying it's not supported anymore and to upgrade to the latest patch release, referencing the issue.

Reply:
Already did in the airlock PR. Please check.